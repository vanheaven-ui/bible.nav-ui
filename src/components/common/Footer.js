import { Link } from 'react-router-dom';
import '../../styles/footer.css';
import logo from '../../images/logo1.png';

const Footer = () => (
  <footer id="footer">
    <div className="footer-wrap">
      <h3 className="footer-logo">
        <img src={logo} alt="logo" width="50px" height="50px" />
        <Link to="/">Bible.nav</Link>
      </h3>
      <p className="address">
        Plot 3, Speke Road street, Kampala Rd 3
        <br />
        Kampala, Uganda.
      </p>
      <ul className="social-links">
        <li>
          <a href="https://www.linkedin.com/in/vanheaven/">
            <i className="fab fa-linkedin" />
          </a>
        </li>
        <li>
          <a href="https://twitter.com/MworekwaE">
            <i className="fab fa-twitter-square" />
          </a>
        </li>
        <li>
          <a href="https://github.com/vanheaven-ui">
            <i className="fab fa-github-square" />
          </a>
        </li>
      </ul>
      <p className="copyright">Bible.nav 2021. All rights reserved</p>
    </div>
  </footer>
);

export default Footer;
