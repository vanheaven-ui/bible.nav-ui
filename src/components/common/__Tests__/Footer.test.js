import React from 'react';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import rootDiv from '../../../testHelper';
import '@testing-library/jest-dom';
import Footer from '../Footer';

it('renders without crashing', () => {
  ReactDOM.render(<Router><Footer /></Router>, rootDiv());
});

it('does not change unexpectedly', () => {
  const tree = renderer.create(<Router><Footer /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders the correct wrapper element', () => {
  const { container } = render(<Router><Footer /></Router>);
  expect(container.firstChild).toHaveAttribute('id', 'footer');
});
