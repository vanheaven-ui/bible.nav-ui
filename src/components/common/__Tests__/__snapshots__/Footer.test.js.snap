// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not change unexpectedly 1`] = `
<footer
  id="footer"
>
  <div
    className="footer-wrap"
  >
    <h3
      className="footer-logo"
    >
      <img
        alt="logo"
        height="50px"
        src="logo1.png"
        width="50px"
      />
      <a
        href="/"
        onClick={[Function]}
      >
        Bible.nav
      </a>
    </h3>
    <p
      className="address"
    >
      Plot 3, Speke Road street, Kampala Rd 3
      <br />
      Kampala, Uganda.
    </p>
    <ul
      className="social-links"
    >
      <li>
        <a
          href="https://www.linkedin.com/in/vanheaven/"
        >
          <i
            className="fab fa-linkedin"
          />
        </a>
      </li>
      <li>
        <a
          href="https://twitter.com/MworekwaE"
        >
          <i
            className="fab fa-twitter-square"
          />
        </a>
      </li>
      <li>
        <a
          href="https://github.com/vanheaven-ui"
        >
          <i
            className="fab fa-github-square"
          />
        </a>
      </li>
    </ul>
    <p
      className="copyright"
    >
      Bible.nav 2021. All rights reserved
    </p>
  </div>
</footer>
`;
